.github/workflows/compliance.yml

```diff
@@ -58,7 +58,7 @@ jobs:
         git config --global user.name "Your Name"
         git remote -v
         # Sauce tag checks before rebasing
-        git log --oneline --first-parent origin/${BASE_REF}..HEAD | grep -E -v "\[nrf (mergeup|fromtree|fromlist|noup)\]" && { echo 'Sauce tag missing'; exit 1; }
+        git log --oneline --first-parent origin/${BASE_REF}..HEAD | grep -E -v "\[nrf (mergeup|fromtree|fromlist|noup)\][ ]" && { echo 'Sauce tag missing'; exit 1; }
```

Comment:
why the `[ ]`?

Reply:
That's how you match a space character

Reply:
I have always used `\ `

Reply:
I find this more readable, but I will change it if you want.

Reply:
No, that's OK. I just found it confusing with the `[nrf ]` before it.